src/ekobox/cli/inspect.py

```diff
@@ -27,7 +27,7 @@ def subcommand(ctx, path: pathlib.Path):
 @click.pass_obj
 def sub_mu2(operator: EKO):
     """Check operator's mu2grid."""
-    rich.print_json(data=operator.mu2grid.tolist())
+    rich.print_json(data=operator.mu2grid)
```

Comment:
Wait, `mu2grid` is still an array, I'd like to really print as JSON (for post-processing, if ever).

Was there a typing problem?

Reply:
apparently - else I wouldn't have touched it (as already said CLI is never used nor tested)

Reply:
It seems like we (i.e. I) changed type to `operator.mu2grid`, but I forgot to update here. Thanks Mypy (and @felixhekhorn, of course)!